tests/specs/obfuscate.e2e.js

```diff
@@ -0,0 +1,92 @@
+import { supportsFetchExtended } from '../../tools/browser-matcher/common-matchers.mjs'
+
+describe('obfuscate rules', () => {
+  withBrowsersMatching(supportsFetchExtended)('should apply to all payloads', async () => {
+    const spaPromise = browser.testHandle.expectEvents()
+    const ajaxPromise = browser.testHandle.expectAjaxEvents()
+    const timingsPromise = browser.testHandle.expectTimings()
+    const errorsPromise = browser.testHandle.expectErrors()
+    const insPromise = browser.testHandle.expectIns()
+    const resourcePromise = browser.testHandle.expectResources()
+    const rumPromise = browser.testHandle.expectRum()
+
+    const config = {
```

Comment:
Should probably move this variable declaration outside the test, above the describe. You can also probably remove all the properties except the `init.obfuscate` properties. The test server handle will merge the properties with the defaults in `tools/wdio/plugins/testing-server/default-asset-query.mjs`.